docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -143,7 +153,7 @@ This property is optional; the default is ``true``.
 Protocol used to communicate with brokers.
 Valid values are: PLAINTEXT, SSL.
 
-This property is optional; default is ``PLAINTEXT``.
+This property is optional; default is ``PLAINTEXT``, but it can be overwritten in ``kafka.config.resources``.
```

Comment:
No, it should not be overwritten by `kafka.config.resource`. In case it is not defined, then one from `kafka.config.resource` should be used.

Reply:
Yes you are right. The current behaviour is following your statement.

Reply:
I'll remove the modified statement.